### PR TITLE
support custom redirect url

### PR DESCRIPTION
**Pull request by nirgur**
fixes: https://github.com/descope/etc/issues/1912

### CODE REVIEW

File: src/lib/components/Descope.tsx [hunk 1]
```diff
@@ -36,13 +37,26 @@ const DescopeWC = lazy(async () => {
 				theme={theme}
 				debug={debug}
 				telemetryKey={telemetryKey}
+				redirect-url={redirectUrl}
```
Comment (asafshen) [inline src/lib/components/Descope.tsx:40]: Let's add it to the readme configuration table 

Also I wonder it we should add an env variable to the example app, but less critical

**Review by asafshen — APPROVED** [review]
Looks good
Small comment

File: src/lib/components/Descope.tsx (hunk 1 above)
Comment (dorsha) [reply src/lib/components/Descope.tsx:40]: Added for both.

Comment (dorsha) [reply src/lib/components/Descope.tsx:40]: Tested and it works good.

**Review by orius123 — APPROVED** [review]

**Pull request merged**